src/utils/exportUtils/exportJS.ts

```diff
@@ -0,0 +1,27 @@
+export default `
```

Comment:
might be good to get this into a separate js file and use raw-loader like above, that way the file will work better in IDEs